modules/dfpAdServerVideo.js

```diff
@@ -249,6 +253,55 @@ function getCustParams(bid, options, urlCustParams) {
   return encodedParams;
 }
 
+export async function getBidVastWithGam(adTagUrl, cacheMap = vastsLocalCache) {
+  const gamAdTagUrl = new URL(adTagUrl);
+  const custParams = new URLSearchParams(gamAdTagUrl.searchParams.get('cust_params'));
+  const videoCacheKey = custParams.get('hb_uuid');
```

Comment:
Id prefer not doing the URL parsing, since it couples this to the URL structure which is also tied to the ad server. Since the local cache generates the UUIDs, it seems like the response parser could look for them without first checking the URL.